packages/example-app-base/src/components/ComponentPage/ComponentPage.tsx

```diff
@@ -49,21 +49,23 @@ export class ComponentPageBase extends React.PureComponent<IComponentPageProps>
   public render() {
     const { componentName, className, otherSections, styles, theme } = this.props;
 
+    const showOnlyExamples = location && location.hash.indexOf('docsExample=true') > -1;
+
     const classNames = (this._styles = getClassNames(styles, { theme }));
 
     return (
       <div className={css(classNames.root, className)}>
         <div className={componentName}>
-          {this._getPageHeader()}
+          {!showOnlyExamples && this._getPageHeader()}
           <div className={classNames.body}>
-            {this._getComponentStatusBadges()}
-            {this._getOverview()}
-            {this._getBestPractices()}
+            {!showOnlyExamples && this._getComponentStatusBadges()}
```

Comment:
Is there any way you can avoid separately checking `!showOnlyExamples` on every line? Probably not a perf issue or anything, more just readability (arguably).

Reply:
I will clean up a code as soon as I have a working prototype to test in docs.

Reply:
Fair enough!